components/suggestion/at_mention_provider/at_mention_suggestion.tsx

```diff
@@ -33,7 +33,11 @@ interface Group extends Item {
     member_count: number;
 }
 
-class AtMentionSuggestion extends Suggestion {
+type Props = {
+    intl: IntlShape;
+}
+
+class AtMentionSuggestion extends Suggestion<Props> {
```

Comment:
Should we be passing a second type parameter to `Suggestion` like `Suggestion<Props, Item>`?